extensions/php/src/features/validationProvider.ts

```diff
@@ -147,6 +150,11 @@ export default class PHPValidationProvider {
 			let shellSettings = section.get<any>('validate.runInShell');
 			if (typeof(shellSettings) === 'boolean') {
 				this.runInShell = shellSettings;
+				if (this.platform.toLowerCase() === 'win32') {
+					this.shellExecutable = 'C:\\Windows\\sysnative\\bash.exe';
```

Comment:
I think we can't assume that a user wants the bash if runInShell is set to true. I would expect that the user wants to use what is defined in ComSpec under Windows.

And then we need to set the shellArgs accordingly as well since -c will not work with cmd.exe nor with powershell.exe. For cmd.exe this is /C and for PowerShell it is /Command